components/brave_rewards/resources/ui/reducers/wallet_reducer.ts

```diff
@@ -266,6 +263,30 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
       chrome.send('brave_rewards.removeAllPendingContribution')
       break
     }
+    case types.GET_BALANCE: {
+      chrome.send('brave_rewards.fetchBalance')
+      break
+    }
+    case types.ON_BALANCE: {
+      const status = action.payload.status
+      let ui = state.ui
+      if (status === 1) {
```

Comment:
Same comment as above; would be good to have a clearer idea of what the various status codes represent, if possible.

Reply:
done